msal_extensions/cache_lock.py

```diff
@@ -3,6 +3,7 @@
 import sys
 import errno
 import portalocker
+from distutils.version import LooseVersion
```

Comment:
Interesting. I did not know this. The `distutils.version` was [barely mentioned but not documented in Python's doc](https://docs.python.org/3/distutils/apiref.html#module-distutils.version). With that specific name, I managed to find  [doc for `LooseVersion`](http://epydoc.sourceforge.net/stdlib/distutils.version.LooseVersion-class.html), and accidentally saw [this issue report](https://github.com/postmarketOS/pmbootstrap/issues/342).

Anyway, we can go with this, for now.

#LearnSomethingNewEveryday

Reply:
The alternative is setuptools, which is less hacky but adds a new runtime dependency

Reply:
Agree with your choice to avoid yet another dependency.